common/components/controllers/AboutProjectController.jsx

```diff
@@ -203,7 +203,7 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
                   </Grid>
               <div className="AboutProjects_tabs">
                 <a onClick={() => this.changeHighlighted('details')} className={this.state.tabs.details ? 'AboutProjects_aHighlighted' : 'none'}href="#project-details">Details</a>
-                <a onClick={() => this.changeHighlighted('skills')} className={this.state.tabs.skills ? 'AboutProjects_aHighlighted' : 'none'} href="#skills-needed">Skills Needed</a>
+                {project && !_.isEmpty(project && project.project_positions) && <a onClick={() => this.changeHighlighted('skills')} className={this.state.tabs.skills ? 'AboutProjects_aHighlighted' : 'none'} href="#skills-needed">Skills Needed</a>}
```

Comment:
```suggestion
                {project && !_.isEmpty(project.project_positions) && <a onClick={() => this.changeHighlighted('skills')} className={this.state.tabs.skills ? 'AboutProjects_aHighlighted' : 'none'} href="#skills-needed">Skills Needed</a>}
```

Reply:
You're already checking project is defined, so no need to re-check it inside _.isEmpty